packages/site-components/src/SearchInput/index.tsx

```diff
@@ -11,6 +11,7 @@ export function SearchInput() {
   const { searchIndex } = useSearchIndex();
   const [searchTerm, setSearchTerm] = useState('');
   const [searchResults, setSearchResults] = useState<SearchResults>([]);
+  const [searchOverflow, setSearchOverflow] = useState<number>(0);
```

Comment:
I don't think this needs to be in state, derive the overflow from the `searchResults` state